app/templates/components/public/ticket-list.hbs

```diff
@@ -1,5 +1,227 @@
+              <div class="field">
+                <UiDropdown
+                  @class="compact selection"
+                  @forceSelection={{false}}
+                  @onChange={{action "updateOrder" ticket}} as |execute mapper|>
+                  <Input
+                    @type="hidden"
+                    @id={{concat ticket.id "_quantity"}}
+                    @value={{ticket.orderQuantity}} />
+                  <i class="dropdown icon"></i>
+                  <div class="default text">{{if (eq this.tickets.length 1) (max ticket.minOrder 1) 0}}</div>
+                  <div class="menu">
+                    <div class="item" data-value="{{map-value mapper 0}}">0</div>
+                    {{#each (range (max ticket.minOrder 1) (add ticket.maxOrder 1)) as |count|}}
+                      <div class="item" data-value="{{map-value mapper count}}">{{count}}</div>
+                    {{/each}}
+                  </div>
+                </UiDropdown>
+              </div>
+            </td>
+          </tr>
+        {{!-- ticket Subtotal --}}
+          <tr>
+            <th class="ui left aligned two wide">{{t 'Subtotal'}}</th>
+            <td id='{{ticket.id}}_subtotal' class="ui left aligned">
+              <Orders::TicketPrice
+                @ticket={{ticket}}
+                @currency={{this.eventCurrency}}
+                @amount={{ticket.subTotal}} />
+            </td>
+          </tr>
+
+      </tbody>
+      {{/unless}}
+    {{/each}}
+          <tfoot class="full-width">
+      <tr>
+        <td colspan="3">
+            <table class="ui small" style="float: right;">
+              <tbody>
+                {{#if orderAmount.tax}}
+                  <tr><td>{{t 'Sub-Total'}}</td><td><CurrencyAmount @currency={{this.eventCurrency}} @amount={{this.orderAmount.sub_total}}/></td></tr>
+                  <tr>
+                    <td>
+                      <div>{{t 'Tax'}} ({{this.orderAmount.tax.percent}}% {{this.orderAmount.tax.name}})</div>
+                      {{#if orderAmount.tax.included}}
+                        <small class="ui gray-text small">Included in Total</small>
+                      {{/if}}
+                    </td>
+                    <td><CurrencyAmount @currency={{this.eventCurrency}} @amount={{this.orderAmount.tax.amount}}/></td>
+                  </tr>
+                {{/if}}
+                <tr><td>{{t 'Total'}}</td>
+                  {{#if this.amountEditable}}
+                    <td>{{#if this.overridenAmount}}{{currency-symbol this.eventCurrency}}{{/if}}</td><td><Input @value={{this.overridenAmount}} placeholder="Total" /></td>
+                  {{else}}
+                    <td><CurrencyAmount @currency={{this.eventCurrency}} @amount={{this.orderAmount.total}}/></td>
+                  {{/if}}
+                </tr>
+              </tbody>
+            </table>
+        </td>
+      </tr>
+    </tfoot>
+    </table>
+
+{{!-- order now --}}
+    {{#unless this.amountEditable}}
```

Comment:
is it needed to repeat code for order button, please remove this. Keep it out side if else.

Reply:
Ok, I'll do the changes.